pyartcd/pyartcd/pipelines/prepare_release_konflux.py

```diff
@@ -1243,6 +1276,9 @@ async def verify_payload(self):
         """
         Verify that the swept builds match the imagestreams that were updated during build-sync.
         """
+        if self.updated_assembly_group_config.get("operator_index_mode") == "pre-release":
```

Comment:
instead of this, i would suggest adding "if kind:image shipment is being prepared -> run verify, skip if not"